### PR TITLE
Remove final mentions of .responsive

### PR DESCRIPTION
**Pull request by Ianfeather**

### CODE REVIEW

File: app/assets/stylesheets/components/_banner.sass [hunk 1]
```diff
@@ -192,10 +189,8 @@
   position: absolute
   right: 0
 
-  .responsive &
-    float: none
-    +respond-to(medium-view)
-      float: right
+  +respond-to(medium-view)
+    float: right
```
Comment (i-like-robots) [inline app/assets/stylesheets/components/_banner.sass]: It's positioned absolutely so float is redundant

File: app/assets/stylesheets/components/_stack.sass [hunk 2]
```diff
@@ -12,23 +12,16 @@
...
-      max-width: 100%
-      width: 100%
-      +respond-to(wide-view)
-        clear: none
-        display: table-cell
-        margin-left: 10px
-        max-width: 798px
-        width: 798px
+    clear: both
+    width: 100%
```
Comment (i-like-robots) [inline app/assets/stylesheets/components/_stack.sass]: I don't think the width declaration is adding anything here

**Comment by Ianfeather** [comment]
Good spots, thanks. Good to go now?

File: app/assets/stylesheets/components/_gallery.sass [hunk 3]
```diff
@@ -81,15 +81,13 @@ $footer-height: 250px
...
   padding-bottom: 10px
-  .responsive &
-    +font-size(20)
-    +respond-to(768)
-      +font-size(22)
-    +respond-to(1000)
-      +font-size(24)
+  +respond-to(768)
+    +font-size(22)
+  +respond-to(1000)
```
Comment (i-like-robots) [inline app/assets/stylesheets/components/_gallery.sass]: Should we just use our standard BP names here? Not much in it. Doesn't matter for this PR I guess.

**Comment by i-like-robots** [comment]
Looks solid to me :+1:

**Pull request merged**